api/routes/routes.go

```diff
@@ -16,9 +17,13 @@ func Router(db *sql.DB) *mux.Router {
 	adminHandler := admin.NewHandler(adminStore)
 	adminHandler.RegisterRoutes(subrouter)
 
-	sockStore := inventory.NewSockStore(db) 
+	sockStore := inventory.NewSockStore(db)
 	sockHandler := inventory.NewSockHandler(sockStore)
 	sockHandler.RegisterRoutes(subrouter, adminStore)
 
+	orderStore := orders.NewOrderStore(db)
+	orderHandler := orders.NewHandler(orderStore, adminStore)
```

Comment:
Since we don't need the `adminStore` outside of the `orderHandler.RegisterRoutes`, instead of passing it here just pass it there instead like this: `orderHandler.RegisterRoutes(subrouter, adminStore)`